src/ds5/ds5-device.cpp

```diff
@@ -464,6 +464,11 @@ namespace librealsense
                     RS2_OPTION_ASIC_TEMPERATURE));
         }
 
+        if (_fw_version >= firmware_version("5.9.10.0"))
```

Comment:
I'm not sure that at the moment we can use >= and not ==.
Only once it is in the mainline firmware, we can add >=, otherwise there is a chance we will get firmware 5.10.x that does not support this feature.